sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -937,6 +937,15 @@ object SQLConf {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefault(0L)
 
+  val THRIFTSERVER_FORCE_CANCEL =
+    buildConf("spark.sql.thriftServer.interruptOnCancel")
+      .doc("When true, if a running query has been cancelled then all running tasks will be" +
+        "interrupted. When false, if a running query has been cancelled then all running task " +
```

Comment:
How about this update?
```
      .doc("When true, all running tasks will be interrupted if one cancels a query. When false, " +
        "all running tasks will be remained until finished.")
```

Reply:
Yea, I think it sounds more close to the meaning of `interruptOnCancel`.

Reply:
Updated.